### ISSUE TITLE
File names?

### ISSUE THREAD

**Issue by narcolepticinsomniac** [ugetdm/uget-integrator#59]
I'm not sure if this is the integrator, or some limitation in uGet. The issue is either that file names are not getting passed along to uGet, or uGet ignores them.  If it's the latter, feel free to just close this issue.

I always kinda thought file naming was funky, but recently I was experimenting with the `html` download attribute. `download="vid.mp4"` for example, if I pause uGet integrator, the specified name is passed on to the save-as dialog, but if the integrator is enabled, the file name passed on to uGet is whatever random string grabbed from the URL.

If it is the integrator, it'd be great if you could pass along the specified file name.

**Comment by fagnerpatricio** [comment]
I Have a similar issue.

**Comment by slavikme** [comment]
Agree. This is pretty annoying that I need to change the filename manually every time.

**Comment by mquan86** [comment]
I followed instruction here and it works: https://aavtech.site/2019/08/workaround-for-uget-integration-wrong-file-name/

> In uget-integrator.py find the line.
> 
> fileName = basename(urlparse(url).path).replace('%20', ' ')
> Comment out the line and add a new line as shown below
> 
> #fileName = basename(urlparse(url).path).replace('%20', ' ')
> fileName = ""

If that article is true, then uget-integrator should change the way it handles file name and let the back end handle it. Or at least has optional to turn on/off.
Thanks.

**Comment by junglized** [comment]
I see that never's been addressed....
